jupyter_lidar_inspection.ipynb

```diff
@@ -0,0 +1,918 @@
+    "series.show()\n",
+    "\"series.save(\"/your/path/image.gif\") # You can export that into a GIF, add it to your presentation and be cool"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3b999c1",
+   "metadata": {},
+   "source": [
+    "Set a smaller computational region for checking how would we like the data processed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b572a576",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gs.run_command(\"g.region\",save='my_region',flags=\"s\") # Let's save our region in the GRASS way\n",
+    "region_dict = gs.parse_command(\"g.region\",flags=\"gu\") # Tthe parse_command parses the region parameters into a dictionary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "90b07d6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a copy of the original region dictionary\n",
+    "smaller_region_dict = region_dict.copy()\n",
+    "\n",
+    "# Update specific keys in the new dictionary\n",
+    "# Set the values of the extent you want to examine\n",
+    "updates = {\n",
+    "    'n': '79000',\n",
+    "    's': '78750',\n",
+    "    'w': '478250',\n",
+    "    'e': '478500'\n",
+    "}\n",
+    "# Apply the updates to the new dictionary\n",
+    "smaller_region_dict.update(updates)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d7839235-f132-49bf-bee0-fc2490f07f09",
+   "metadata": {},
+   "source": [
+    "Let's update momentarily our region so we can visualize a smaller portion of it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe7dc980",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n=int(smaller_region_dict['n'])\n",
+    "s=int(smaller_region_dict['s'])\n",
+    "w=int(smaller_region_dict['w'])\n",
+    "e=int(smaller_region_dict['e'])\n",
+    "univar_stat=gs.parse_command('r.univar',map='point_cloud_1m', flags='g')\n",
+    "z_min=float(univar_stat['min'])\n",
+    "z_max=float(univar_stat['max'])\n",
+    "\n",
+    "gs.run_command(\"g.region\",\n",
+    "               save='smaller_region',\n",
+    "               n=n,\n",
+    "               s=s,\n",
+    "               w=w,\n",
+    "               e=e,\n",
+    "               flags=\"d\")\n",
+    "gs.run_command(\"g.region\",region='smaller_region')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5b70bf8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gs.run_command(\"g.region\",\n",
+    "               save='smaller_region',\n",
+    "               n=int(smaller_region_dict['n']),\n",
+    "               s=int(smaller_region_dict['s']),\n",
+    "               w=int(smaller_region_dict['w']),\n",
+    "               e=int(smaller_region_dict['e']),\n",
+    "               flags=\"d\")\n",
+    "gs.run_command(\"g.region\",region='smaller_region')"
+   ]
```

Comment:
i have the feeling all this region management can be simplified a bit, what do you say?